app/uk/gov/hmrc/helptosavefrontend/controllers/EligibilityCheckController.scala

```diff
@@ -137,10 +138,11 @@ class EligibilityCheckController @Inject() (val messagesApi:             Message
           auditor.sendEvent(new EligibilityCheckEvent(appName, nino, Some(IneligibilityReason.AccountAlreadyOpened.legibleString)))
 
           // set the ITMP flag here but don't worry about the result
-          helpToSaveService.setITMPFlag(nino).fold(
-            e ⇒ logger.warn(s"Could not set ITMP flag for user $nino: $e"),
-            _ ⇒ logger.info(s"Set ITMP flag for user $nino")
-          )
+          helpToSaveService.setITMPFlag(nino).value.onComplete{
+            case Failure(e)        ⇒ logger.warn(s"For NINO [$nino]: Could not set ITMP flag, future failed: ${e.getMessage}")
+            case Success(Left(e))  ⇒ logger.warn(s"For NINO [$nino]: Could not set ITMP flag: $e")
+            case Success(Right(_)) ⇒ logger.info(s"For NINO [$nino]: Successfully set ITMP flag for user")
+          }
 
           Ok("You've already got an account - yay!!!")
```

Comment:
Do we need to chat about what the final msg will be here?

Reply:
this is a stub page - this will actually be a redirect to NS&I but we don't have that yet so this is a placeholder page